tests/install/test.py

```diff
@@ -97,26 +151,42 @@ def test_install(self, firmware, create_vms, iso_remaster, iso_version, local_sr
             assert len(ips) == 1
             host_vm.ip = ips[0]
 
-            host_vm.ssh(["ls"])
-            logging.debug("ssh works")
-
-            # wait for "yum install" phase to finish
-            wait_for(lambda: host_vm.ssh(["grep",
-                                          "'DISPATCH: NEW PHASE: Completing installation'",
-                                          "/tmp/install-log"],
-                                         check=False, simple_output=False,
-                                         ).returncode == 0,
-                     "Wait for rpm installation to succeed",
-                     timeout_secs=40*60) # FIXME too big
-
-            # wait for install to finish
-            wait_for(lambda: host_vm.ssh(["grep",
-                                          "'The installation completed successfully'",
-                                          "/tmp/install-log"],
-                                         check=False, simple_output=False,
-                                         ).returncode == 0,
-                     "Wait for system installation to succeed",
-                     timeout_secs=40*60) # FIXME too big
+            if source_type == "pxe":
+                # When using PXE boot the IP is available but ssh is not so wait...
+                while True:
+                    try:
+                        host_vm.ssh(["ls"])
+                    except commands.SSHCommandFailed as e:
+                        if e.returncode == 255:
+                            logging.debug("ssh connexion refused, wait 10s ...")
+                            time.sleep(10)
```

Comment:
We have a short idiom to wait for ssh port to be open first, that may be sufficient?